.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
Should it be `/.idea/`?

Reply:
I used .idea/ to ignore the .idea folders in all subpaths as well.
If I used /.idea/ it would only ignore it in the root.

by using .idea/ to ignore all folders irregardless of depth with the name .idea
we could remove integration/.idea/ from the .gitignore file.

Reply:
Make sense.

Then we can remove another `.idea/` line below.

Reply:
Also, keep it in a separate section by itself, better add a comment too. So it will not asked about again.

Reply:
Good idea, will amend the commit & pull request with your suggestion.

Reply:
Is it how it works? Seems it doesn't work for me for other directories.